### PR TITLE
Adds new donor items, a new public hairstyle, and private donor items for Thedragmeme

### PR DESCRIPTION
**Pull request by Draggeru**
## About The Pull Request
This PR adds an entirely new hairstyle: Royal Curls

In addition, it adds donator items for myself (The dress, heels, blindfold, and cloak) and adds donor items for other donors.

stuff

## Changelog
:cl:
add: adds some donator items
add: Adds a new hairstyle, royal curls
:cl:

### CODE REVIEW

File: code/modules/mob/dead/new_player/sprite_accessories.dm [hunk 1]
```diff
@@ -655,6 +655,10 @@
 	name = "Ronin"
 	icon_state = "hair_ronin"
 
+/datum/sprite_accessory/hair/royal_curls
```
Comment (Cyprex) [inline code/modules/mob/dead/new_player/sprite_accessories.dm:658]: needs modularization, but I'm happy to do that for you!

Comment (Draggeru) [reply code/modules/mob/dead/new_player/sprite_accessories.dm:658]: Which folder would you suggest the hairstyle goes in?

**Comment by Draggeru** [comment]
I decided a shawl looks better than the cloak
EASgZXPAB3gy9evkARASEAdNBCKAYgX8Cx4aAEa4k0IAAAAASUVORK5CYII

**Review by nevimer — CHANGES_REQUESTED** [review]
Literally anything touching a file in modular_skyrat folders will be overrwritten, especially in .dmi files as they cannot be merged easily.

File: code/modules/mob/dead/new_player/sprite_accessories.dm [hunk 2]
```diff
@@ -2220,7 +2220,7 @@
 /datum/sprite_accessory/moth_wings/plasmafire
 	name = "Plasmafire"
 	icon_state = "plasmafire"
-	
+
```
[the remark this answers is not in the document]
Comment (Draggeru) [reply code/modules/mob/dead/new_player/sprite_accessories.dm:2223]: I used that file as a reference file, I have no idea why it's doing that.

[the remark this answers is not in the document]
Comment (Higgin) [reply code/modules/mob/dead/new_player/sprite_accessories.dm:2223]: There was a tab mistakenly left on line 2223 between the plasmafire wings and the moffra wings from what I can see.

**Comment by nevimer** [comment]
Just a note that you can have the in hand icon changes, in a new file for that object. It's just that if upstream touches icon files we edit, we lose the icons we added or changed.

**Comment by Higgin** [comment]
Yeah - I don't know why the modular SR balls file was getting touched, though, it had nothing to do with the inhands these new pieces are calling for.

**Comment by Draggeru** [comment]
> Just a note that you can have the in hand icon changes, in a new file for that object. It's just that if upstream touches icon files we edit, we lose the icons we added or changed.

All I did was open the file for the ball sprites, I was using it as a reference to make sure the custom in hands for this set was correct. After I closed it VSCode and GitHub are insisting I made changes when I didn't.

The only changes/additions that were actually made are for the items listed in the PR description.

**Comment by Higgin** [comment]
@nevimer The modular_skyrat file should no longer be getting touched.

**Pull request merged**